<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>585</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Resources/Inspector.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>141</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>141</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>141</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Analyze</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="summaryTab">
       <attribute name="title">
        <string>Summary</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListView" name="listView">
          <property name="font">
           <font>
            <family>Courier New</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dosHeaderTab">
       <attribute name="title">
        <string>DOS Header</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="dosHeaderTableWidget">
          <row>
           <property name="text">
            <string>e_magic</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_cblp</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_cp</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_crlc</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_cparhdr</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_minalloc</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_maxalloc</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_ss</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_sp</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_csum</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_ip</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_cs</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_lfarlc</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_ovno</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_oemid</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_oeminfo</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>e_lfanew</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Offset</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
          <item row="0" column="3">
           <property name="text">
            <string>Magic number</string>
           </property>
          </item>
          <item row="1" column="3">
           <property name="text">
            <string>Bytes on last page of file</string>
           </property>
          </item>
          <item row="2" column="3">
           <property name="text">
            <string>Pages in file</string>
           </property>
          </item>
          <item row="3" column="3">
           <property name="text">
            <string>Relocations</string>
           </property>
          </item>
          <item row="4" column="3">
           <property name="text">
            <string>Size of header in paragraphs</string>
           </property>
          </item>
          <item row="5" column="3">
           <property name="text">
            <string>Minimum extra paragraphs needed</string>
           </property>
          </item>
          <item row="6" column="3">
           <property name="text">
            <string>Maximum extra paragraphs needed</string>
           </property>
          </item>
          <item row="7" column="3">
           <property name="text">
            <string>Initial (relative) SS value</string>
           </property>
          </item>
          <item row="8" column="3">
           <property name="text">
            <string>Initial SP value</string>
           </property>
          </item>
          <item row="9" column="3">
           <property name="text">
            <string>Checksum</string>
           </property>
          </item>
          <item row="10" column="3">
           <property name="text">
            <string>Initial IP value</string>
           </property>
          </item>
          <item row="11" column="3">
           <property name="text">
            <string>Intial (relative) CS value</string>
           </property>
          </item>
          <item row="12" column="3">
           <property name="text">
            <string>File address of relocation table</string>
           </property>
          </item>
          <item row="13" column="3">
           <property name="text">
            <string>Overlay number</string>
           </property>
          </item>
          <item row="14" column="3">
           <property name="text">
            <string>OEM identifier</string>
           </property>
          </item>
          <item row="15" column="3">
           <property name="text">
            <string>OEM information</string>
           </property>
          </item>
          <item row="16" column="3">
           <property name="text">
            <string>File address of new exe header</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ntHeadersTab">
       <attribute name="title">
        <string>NT Headers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="ntHeadersTableWidget">
          <row>
           <property name="text">
            <string>Signature</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Offset</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fileHeaderTab">
       <attribute name="title">
        <string>File Header</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="fileHeaderTableWidget">
          <row>
           <property name="text">
            <string>Machine</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>TimeDateStamp</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>PointerToSymbolTable</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>NumberOfSymbols</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Characteristics</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Offset</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="optionalHeaderTab">
       <attribute name="title">
        <string>Optional Header</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTableWidget" name="optionalHeaderTableWidget">
          <row>
           <property name="text">
            <string>Magic</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MajorLinkerVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MinorLinkerVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfCode</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfInitializedData</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfUninitializedData</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>AddressOfEntryPoint</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>BaseOfCode</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>BaseOfData</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>ImageBase</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SectionAlignment</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>FileAlignment</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MajorOperatingSystemVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MinorOperatingSystemVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MajorImageVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MinorImageVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MajorSubsystemVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>MinorSubsystemVersion</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Win32VersionValue</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfImage</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfHeaders</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>CheckSum</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Subsystem</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>DllCharacteristics</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfStackReserve</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfStackCommit</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfHeapReserve</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>SizeOfHeapCommit</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>NumberOfRvaAndSizes</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Offset</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataDirectoriesTab">
       <attribute name="title">
        <string>Data Directories</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTableWidget" name="dataDirectoriesTableWidget">
          <row>
           <property name="text">
            <string>Export Directory Address</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Import Directory Address</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Offset</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sectionHeaderTab">
       <attribute name="title">
        <string>Section Headers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QListWidget" name="sectionHeaderListWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dllImportTab">
       <attribute name="title">
        <string>DLL Imports</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QListWidget" name="dllImportListWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionToggleDarkMode"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToggleDarkMode">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WeatherClear"/>
   </property>
   <property name="text">
    <string>Toggle dark mode</string>
   </property>
   <property name="toolTip">
    <string>Toggle dark/light mode</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
